server/message_hooks.go

```diff
@@ -466,7 +466,7 @@ func (p *Plugin) handlePromptForConnection(userID, channelID string) {
 		p.API.LogDebug("Unable to get the last prompt timestamp for the channel", "ChannelID", channelID, "Error", err.Error())
 	}
 
-	if time.Until(timestamp) < -time.Hour*24*30 {
+	if time.Until(timestamp) < -time.Hour*time.Duration(p.getConfiguration().PromptIntervalForDMsAndGMs) {
```

Comment:
I think if the value is 0 here we will get the prompt every time a user messages